README.md

```diff
@@ -893,12 +893,14 @@ repos:
     rev: stable
     hooks:
       - id: black
+        args: [--exclude, '/(\.git|\.tox|\.venv|build|static|dist|node_modules)/']
```

Comment:
well for one this arg is completely unnecessary -- pre-commit only runs on checked in files
for two, this should not be in the default configuration -- people copy paste this blindly and we certainly don't want to cargo-cult this everywhere

Reply:
That's a good point, thanks! Will come up with an alternative.

It's not completely unnecessary though... I initially wrote this because I was writing a pre-commit based Travis CI lint job. It invokes `pre-commit run --all-files`, but without adding a pyproject.toml configuration for the sole purpose of specifying an excluded set of files.

Other people may have other needs, such as specifying a target folder `src/`.

I'll try adding this as a separate suggestion.

Reply:
those wouldn't be via args, that would be `files:` or `exclude:`

Reply:
Roger, didn't know pre-commit had those global options.

Reply:
the docs are pretty thorough in showing all the options: https://pre-commit.com/#adding-pre-commit-plugins-to-your-project

Reply:
Thanks for the advice, I have now referenced the pre-commit docs.

Re: "thorough". It doesn't matter how thorough another project's docs are if the reader doesn't get the idea to look there ;)